applications/RANSModellingApplication/custom_utilities/rans_check_utilities.h

```diff
@@ -0,0 +1,214 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+
+#if !defined(KRATOS_RANS_APPLICATION_CHECK_UTILITIES_H_INCLUDED)
+#define KRATOS_RANS_APPLICATION_CHECK_UTILITIES_H_INCLUDED
+
+#include <string>
+
+#include "containers/model.h"
+#include "includes/checks.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+class RansCheckUtilities
```

Comment:
why don't you extend the check in the core? This is not rans specific
Could also be done in a future PR, just wanted to point it out

Reply:
I will :)